x-pack/test/functional/apps/transform/creation_index_pattern.ts

```diff
@@ -162,6 +181,13 @@ export default function ({ getService }: FtrProviderContext) {
                     field: 'customer_phone',
                   },
                 },
+                aggs: {
+                  'products.discount_amount.max': {
+                    max: {
+                      feild: 'products.discount_amount',
```

Comment:
Typo in `field` (although this part of the expected `pivotAdvancedEditorValue` is currently not checked).

Reply:
fixed in https://github.com/elastic/kibana/pull/68306/commits/7f51f7b832c1569b2ef6bb1da75e1d65996c15eb